### DISCUSSION TITLE
Command 'lets' not found hatası

### DISCUSSION THREAD

**Discussion by HadjerBr** [19/classroom#40] [Anonim] [answered]
![command lets not found](https://user-images.githubusercontent.com/72814758/159015981-06812e8f-33ab-43ab-8c44-cecd2c5ed725.png)
Merhaba, terminalde "lets grad" komutunu çalıştırdığımda "Command 'lets' not found, did you mean:

  command 'less' from deb less (551-1ubuntu0.1)
  command 'lecs' from deb atm-tools (1:2.5.1-4)
  command 'les' from deb atm-tools (1:2.5.1-4)

Try: sudo apt install <deb name>" hatası alıyorum. Ne yapmam lazım?

**Comment by roktas** [comment, marked as answer]
@HadjerBr Classroom kurulumunda sorun var.  [Ödevini tarayıcıda yap](https://roktas.dev/assets/assignment-browser.029cfc44.gif) lütfen.
